lib/plugins/common/json-rpc.js

```diff
@@ -111,10 +112,12 @@ function rpcConfig (cryptoRec) {
   try {
     const configPath = coinUtils.configPath(cryptoRec, BLOCKCHAIN_DIR)
     const config = parseConf(configPath)
+    
     return {
-      username: config.rpcuser,
-      password: config.rpcpassword,
-      port: config.rpcport || cryptoRec.defaultPort
+      username: isRemoteWallet(cryptoRec) ? process.env[`${cryptoRec.cryptoCode}_NODE_USER`] : config.rpcuser,
+      password: isRemoteWallet(cryptoRec) ? process.env[`${cryptoRec.cryptoCode}_NODE_PASSWORD`] : config.rpcpassword,
+      host: isRemoteNode(cryptoRec) ? process.env[`${cryptoRec.cryptoCode}_NODE_HOST`] : 'localhost',
+      port: isRemoteNode(cryptoRec) ? process.env[`${cryptoRec.cryptoCode}_NODE_PORT`] : config.rpcport || cryptoRec.defaultPort
```

Comment:
Aren't all these envvars set in the `.env` file, whether it's a dev or prod environment? If so, shouldn't the envvars be respected, instead of silently ignoring them in a dev environment? Having a fallack default is probably still worthwhile.

Reply:
Not always. as for local node + wallet (the only setup we currently have), those environment variables are not required, hence the check and the fallback values

Reply:
Got it, I must have misread the condition because I was thinking `isDev` the whole time.